src/som/interpreter/actors/EventualMessage.java

```diff
@@ -323,24 +331,40 @@ public static EventualMessage getCurrentExecutingMessage() {
 
   /**
    * Indicates that execution should stop and yield to the debugger,
-   * before the computed value is used to resolve the promise.
+   * before the message is processed.
+   */
+  public boolean haltOnReceive() {
+    return haltOnReceive;
```

Comment:
you can read here: https://docs.oracle.com/javase/tutorial/javabeans/writing/properties.html
Also in the JDK classes you can see that Java programmers follow this naming convention. For example: [Container.java](https://docs.oracle.com/javase/7/docs/api/java/awt/Container.html), [ArrayList.java](https://docs.oracle.com/javase/7/docs/api/java/util/ArrayList.html)